### PR TITLE
Log in form gets user from database and set it as Context

### LINKED ISSUE #64: Task: Link log in page to return a user object from back-end
This task relates to user story #26.

Link log in page to return a user object from back-end.

### PR DESCRIPTION
**Pull request by nolanbastien**
Log in form gets user from database and set it as Context

### CODE REVIEW

File: frontend/src/Components/Layout/Navigation.js [hunk 1]
```diff
@@ -14,12 +21,14 @@ function Navigation() {
...
-          <Nav.Link as={Link} to="/signup">Sign Up</Nav.Link>
-          <Nav.Link as={Link} to="/login">Log In</Nav.Link>
-          <Nav.Link as={Link} to="/">Log Out</Nav.Link>
+        </Nav>
+        <Nav>
+          {userState && <Nav.Link as={Link} to="/myorders">My Orders</Nav.Link> }
+          {userState &&  <Nav.Link as={Link} to="/editprofile">Manage My Profile</Nav.Link> }
+          {!userState &&  <Nav.Link as={Link} to="/signup">Sign Up</Nav.Link> }
+          {!userState && <Nav.Link as={Link} to="/login">Log In</Nav.Link> }
+          {userState && <Nav.Link onClick={userDispatch}>Log Out</Nav.Link> }
```
Comment (Aboudik1452) [inline frontend/src/Components/Layout/Navigation.js:31]: Refreshing the page logs you out. In other words, it resets the state. To implement this, simply replace onClick with href=<the page you want to get to>.

**Review by Joejoe1452 — CHANGES_REQUESTED** [review]
Overall good work. Please add comments to functions to make code reading easier🙂

**Review by Kellyacoca — COMMENTED** [review]
good work! I see that the issue we were having with was related to "cos" so you did good! I also see that you used the const functions that we worked on to get form data from form fields. its good to go!

**Review by Joejoe1452 — APPROVED** [review]
Thank you for fixing the logout

**Pull request merged**